Add icons for Autonity testnets

flofrie opened the pull request.

ligi reviewed (changes requested):
please put your icon in iconDownload so the CI can check it. Seeing some diff here:

```
identify Qmae5oAFpcCG1MEMBwGbNodHAR3wqPBxQkon28AF2SfKTX         09:36:19
Qmae5oAFpcCG1MEMBwGbNodHAR3wqPBxQkon28AF2SfKTX SVG 81x81 81x81+0+0 8-bit sRGB 1824B 0.000u 0:00.000
```

flofrie commented:
@ligi or someone else, please review again; this should all be working now.

flofrie commented:
> please put your icon in iconDownload so the CI can check it. Seeing some diff here:
> 
> ```
> identify Qmae5oAFpcCG1MEMBwGbNodHAR3wqPBxQkon28AF2SfKTX         09:36:19
> Qmae5oAFpcCG1MEMBwGbNodHAR3wqPBxQkon28AF2SfKTX SVG 81x81 81x81+0+0 8-bit sRGB 1824B 0.000u 0:00.000
> ```

@ligi , we've addressed your comment; can you please have another look and review? (If not you, should we be contacting someone else? We're not 100% certain of the process ...)
